test/tools/WebListener/Controllers/GetController.cs

```diff
@@ -17,13 +17,13 @@ public JsonResult Index()
             Hashtable args = new Hashtable();
             foreach (var key in Request.Query.Keys)
             {
-                args.Add(key, string.Join(Constants.HeaderSeparator, Request.Query[key]));
+                args.Add(key, string.Join(Constants.HeaderSeparator, (string)Request.Query[key]));
```

Comment:
I think this should be fixed in Runtime repository. Is there an issue for the pattern?

Reply:
> Could you please report to Runtime repo the issue?
> 
> I see there are:
> https://github.com/dotnet/runtime/issues/101261
> https://github.com/dotnet/runtime/pull/101499
> https://github.com/dotnet/runtime/pull/101457
> 
> @xtqqczze I suggest to revert to 11.0 lang for the test project so that we can merge and don't wait comments from dotnet team.

_Originally posted by @iSazonov in https://github.com/PowerShell/PowerShell/pull/24617#discussion_r1855746026_

Reply:
https://github.com/dotnet/runtime/pull/101499 is labelled `needs-breaking-change-doc-created`

Reply:
Started discussion: https://github.com/dotnet/runtime/discussions/110216

Reply:
Could you please explicitly report about the issue with cast needed? I guess it should be fixed in Runtime.

Reply:
Could you please create an issue in Runtime repo with repro steps?

Reply:
It seems like the issue is with `Microsoft.Extensions.Primitives.StringValues`

Reply:
The issue is a breaking change, resolution is by updating code, e.g. https://github.com/dotnet/aspnetcore/pull/55780/commits/7e353a334d5c1af3bb6f190f3e877a13a985880b

Reply:
> Could you please create an issue in Runtime repo with repro steps?

https://github.com/dotnet/aspnetcore/pull/55010#issuecomment-2059637820

Reply:
Opened issue: https://github.com/dotnet/docs/issues/44102

Reply:
```log
test/tools/WebListener/Controllers/GetController.cs(20,38): error CS0121: The call is ambiguous between the following methods or properties: 'string.Join(string?, params string?[])' and 'string.Join(string?, params ReadOnlySpan<string?>)' [test/tools/WebListener/WebListener.csproj]
test/tools/WebListener/Controllers/GetController.cs(26,41): error CS0121: The call is ambiguous between the following methods or properties: 'string.Join(string?, params string?[])' and 'string.Join(string?, params ReadOnlySpan<string?>)' [test/tools/WebListener/WebListener.csproj]
```